ares/ares/node/debugger/tracer/instruction.hpp

```diff
@@ -28,7 +28,7 @@ struct Instruction : Tracer {
   }
 
   auto address(u64 address) -> bool {
-    address &= (1ull << _addressBits) - 1;  //mask upper bits of address
+    address &= (u128(1) << _addressBits) - 1;  //mask upper bits of address
```

Comment:
Let's avoid 128-bit math if it's not strictly needed.
```suggestion
    address &= ~0ull >> (64 - _addressBits);  //mask upper bits of address
```